app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect, useRef } from "react";
+import {
+  Checkbox,
+  IconButton,
+  InputBase,
+  ButtonBase,
+  Popper,
+  fade,
+  makeStyles,
+  createStyles,
+} from "@material-ui/core";
+import Autocomplete, {
+  AutocompleteCloseReason,
+} from "@material-ui/lab/Autocomplete";
+import { CloseIcon, ExpandMore } from "../../components/Icons";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    popper: {
+      borderRadius: theme.shape.borderRadius,
+      borderWidth: 1,
+      borderStyle: "solid",
+      borderColor: "rgba(0, 0, 0, 0.23)",
+      boxShadow: theme.shadows[3],
+      zIndex: 1,
+      backgroundColor: theme.palette.background.default,
+      marginTop: theme.spacing(1),
+    },
+    button: {
+      borderRadius: theme.shape.borderRadius,
+      borderColor: "rgba(0, 0, 0, 0.23)",
+      borderWidth: 1,
+      borderStyle: "solid",
+      padding: "18.5px 14px",
+      margin: theme.spacing(1, 0),
+      width: "100%",
+      fontSize: "16px",
```

Comment:
Don't we have a font size scale defined in the theme after the updates you've done based off the UI/UX team's notes? We should probably stick with one of those values instead for consistency.

Reply:
```suggestion
      fontSize: "16px",
      fontFamily: "inherit"
```
Also another minor font related thing - I noticed the text in the dropdown uses a different font compared to other fields since `ButtonBase` from Material UI seems to tamper with it. Might want to override that here with "inherit" the font family we use across the app.